### ISSUE TITLE
TWRP 3.0.2-0 build to 3.1.1-0 build time miscalculation on so many devices.

### ISSUE THREAD

**Issue by aravindvnair99** [TeamWin/Team-Win-Recovery-Project#974]
TWRP time calculation is wrong since TWRP 3.0.2-0 build and exists till 3.1.1-0 build on so many devices.

![img_20170607_125157584 1](https://user-images.githubusercontent.com/22199259/26866847-97693826-4b80-11e7-905a-49d321b29ccf.jpg)

Time is supposed to be 12:56.

**Comment by JonnyTech** [comment]
Did you set the timezone in settings?

**Comment by aravindvnair99** [comment]
@JonnyTech Yes, I did. Also, the issue is present on so many devices I tried. If I go to a TWRP version lower than TWRP 3.0.2-0 build, it works perfect.
